### PR TITLE
Requirements styling updates

### PR DESCRIPTION
**Pull request by tcho6319**
### Summary 

This pull request is the styling updates to the Requirements Menu after Shane's feedback. 

Style updates include:
- [x] Settings buttons
- [x] Coloring groups between university, college, major, and minor requirements (each group is a different specific color)
- [x] Downwards arrow icon when it’s not activated and up when it is activated for all arrow icons (never to the side)
- [x] Capitalization of View All College Requirements, etc…
- [x] Semi-bold School Requirements (new Proxima Nova font files)
- [x] Different colors for university, college, and major requirements
- [x] Adjust spacing
- [x] cr. in thin text instead of Credits in semi-bold
- [x] Darker shade of arrow icon
- [x] Filled Requirements heading and description: 80% opacity and smaller font size
- [x] Removed gear icons

### Test Plan 
Testing of accurate designs was done by inspecting element and comparing to Figma. Basic testing was done to ensure functionality remained unchanged (i.e. expand/collapse, etc.)

### CODE REVIEW

File: src/components/Requirements.vue [hunk 1]
```diff
@@ -7,49 +7,56 @@
     <div class="req" v-for="(req, index) in reqs" :key="req.id">
       <div class="row top">
         <p class="name col p-0">{{ req.name }} <span class="specific" v-if="req.specific">({{ req.specific }})</span></p>
+        <img class="pointer" src="@/assets/images/gear.svg" alt="settings gear" />
```
Comment (willespencer) [inline src/components/Requirements.vue:10]: Not going to implement this for now so remove it please.

File: src/components/Requirements.vue [hunk 2]
```diff
@@ -596,12 +624,32 @@ button.view {
...
     font-size: 12px;
     line-height: 12px;
   }
 }
 
+.completed {
+  margin-top: 1rem;
+  &-ptext {
+    font-size: 12px;
+    opacity: 0.8;
```
Comment (willespencer) [inline src/components/Requirements.vue:636]: For completed reqs, are the descriptions supposed to be the same as normal reqs, but just the titles are smaller with less opacity? Looks a bit off to me.

File: src/components/Requirements.vue [hunk 3]
```diff
@@ -265,6 +272,12 @@ export default {
       ],
       requirementsMap: {
         // CS 1110: 'MQR-AS'
+      },
+      // reqGroupColorMap maps reqGroup to an array [<hex color for progress bar>, <color for arrow image>]
+      reqGroupColorMap: {
+        UNIVERSITY: ['92C3E6', 'lightblue'],
+        COLLEGE: ['1AA9A5', 'blue'],
+        MAJOR: ['105351', 'green']
```
Comment (willespencer) [inline src/components/Requirements.vue:280]: Maybe add a minor color in here for longevity? If Shane didn't design one, pm him.

File: src/components/Requirements.vue [hunk 4]
```diff
@@ -230,7 +238,6 @@ export default {
         //   specific: 'AS',
         //   fulfilled: 46,
         //   required: 120,
-        //   color: '105351',
```
Comment (willespencer) [inline src/components/Requirements.vue:233]: Nice job removing it on the comment haha

File: src/components/Requirements.vue (hunk 3 above)
Comment (hanywang2) [reply src/components/Requirements.vue:280]: In which version of the designs are the color schemes for those req groups? I made a commit with the colors from the 'Sidebar' board on Figma

File: src/components/Requirements.vue (hunk 2 above)
Comment (hanywang2) [reply src/components/Requirements.vue:637]: Shot a msg to Shane in the cp-designs channel

File: src/components/Requirements.vue (hunk 3 above)
Comment (tcho6319) [reply src/components/Requirements.vue]: I messaged Shane about the color for University but he didn't give me a color yet. I think for now the courseplan logo's fourth unused color could work!

File: src/components/Requirements.vue (hunk 2 above)
Comment (tcho6319) [reply src/components/Requirements.vue:637]: Shane got back to me about this and yes the description should be like the completed req titles and have smaller font and lower opacity

File: src/components/r.vue [hunk 5]
```diff
@@ -0,0 +1,759 @@
+<template v-if="semesters">
```
Comment (SamChou19815) [inline src/components/r.vue:1]: What is this? It looks like a leftover of merge conflicts.

Comment (tcho6319) [reply src/components/r.vue]: Yes it was a leftover merge conflict! Deleted r.vue

**Comment by hanywang2** [comment]
Fixing up the seasons emojis and adding them to semesters. Emily pointed that out

**Comment by hanywang2** [comment]
Everything is good now @willespencer

**Pull request merged**